content/webapp/components/SeasonsHeader/index.tsx

```diff
@@ -33,7 +35,8 @@ type Props = {
 };
 
 const SeasonsHeader: FunctionComponent<Props> = ({ season }) => {
-  const { title, standfirst, start, end, labels } = season;
+  const { sliceMachine } = useToggles();
+  const { title, standfirst, start, end, labels, originalStandfirst } = season;
```

Comment:
Make ticket to put `standfirst` on `customtype` rather than in the `slice`

Reply:
[#10753](https://github.com/wellcomecollection/wellcomecollection.org/issues/10753)